tests/queries/0_stateless/02494_query_cache_normalize_ast.sql

```diff
@@ -27,3 +27,34 @@ WHERE type = 'QueryFinish'
   AND query = 'SELECT 1 SETTINGS use_query_cache = true, enable_writes_to_query_cache = false, max_threads = 16;';
 
 SYSTEM DROP QUERY CACHE;
+
+SELECT '--- ---';
```

Comment:
We should add new tests for every new change instead of modifying existing tests.